osbuild/objectstore.py

```diff
@@ -283,13 +283,16 @@ def init(self):
         self._root = self.store.tempdir(prefix="host")
 
         root = self._root.name
-        # Create a bare bones root file system
-        # with just /usr mounted from the host
+        # Create a bare bones root file system with just
+        # /usr and /etc mounted from the host
+        etc = os.path.join(root, "etc")
+        os.makedirs(etc)
         usr = os.path.join(root, "usr")
         os.makedirs(usr)
 
         # ensure / is read-only
         mount(root, root)
+        mount("/etc", etc)
```

Comment:
We discussed this and we think it's better to minimise the footprint of the change by bind mounting only `/etc/containers` here with a note that it's meant to enable running `skopeo` in the root tree.

It would be great if the default policy was installed in `/usr`, but based on the discussion on the issue, it seems that might not be happening.

Reply:
done!